app/Kconfig

```diff
@@ -248,6 +248,39 @@ config ZMK_RGB_UNDERGLOW_ON_START
 #ZMK_RGB_UNDERGLOW
 endif
 
+menuconfig ZMK_BACKLIGHT
+	bool "LED backlight"
+	select PWM
+	select LED
+    select LED_PWM
+
+if ZMK_BACKLIGHT
+
+config ZMK_BACKLIGHT_LEVELS
+	int "Number of brightness levels"
+    range 2 100
+	default 6
+
+config ZMK_BACKLIGHT_LEVEL_START
+	int "Default brightness level"
+    range 1 ZMK_BACKLIGHT_LEVELS
+	default 3
+
+config ZMK_BACKLIGHT_ON_START
+	bool "Default backlight state"
+	default y
+
+config ZMK_BACKLIGHT_SLEEP
```

Comment:
Thoughts on an option for idle too?

Reply:
I see now in the code that this lumps idle/sleep together. This should probably let the user know it's on idle too, or it could be split up.